github-intro/tictactoe.css

```diff
@@ -4,4 +4,10 @@ table {
 
 td {
     border: 3px solid gray;
+    min-width: 50px;
+    min-height: 50px;
```

Comment:
⚠️ This might not be correct. Try `height` instead of `min-height`?

Reply:
I fixed this, thanks for the review!